.github/workflows/semantic-pr-title.yml

```diff
@@ -11,6 +11,6 @@ jobs:
   semantic:
     runs-on: ubuntu-latest
     steps:
-      - uses: amannn/action-semantic-pull-request@v3.0.0
+      - uses: amannn/action-semantic-pull-request@v5
```

Comment:
`v5` is the latest major version of [`amannn/action-semantic-pull-request`](https://github.com/amannn/action-semantic-pull-request/) at the time of this writing.